### ISSUE TITLE
determine mandatory to implement information elements

### ISSUE THREAD

**Issue by djhaynes** [sacmwg/draft-ietf-sacm-information-model#65]
At some point, we need to determine the mandatory to implement information elements that need to be present in data models.  However, it was suggested that we wait on selecting mandatory to implement information elements.

https://www.ietf.org/mail-archive/web/sacm/current/msg03897.html

**Comment by adammontville** [comment]
I agree.  At IETF 97 I'll share information about (provided I don't get booted off the agenda) our dataset, which should help narrow this field a bit.

**Comment by djhaynes** [comment]
Thanks Adam.  I think having some real-world input on what IEs are important would be helpful.
